docs/man/docker.1.md

```diff
@@ -292,6 +295,18 @@ for data and metadata:
       --storage-opt dm.metadatadev=/dev/vdc \
       --storage-opt dm.basesize=20G
 
+# EXEC DRIVER OPTIONS
+
+Options to exec backend can be specified with **--exec-opt** flags. The
+only backend which currently takes options is *native* (libcontainer). Therefore
+use these flags with **-s=**native.
```

Comment:
mmm, I think you're missing some `the`'s, and elsewhere we call it `exec-driver` 

```
Options to the exec-driver can be specified using the **--exec-opt** flags. The
only driver accepting options is the *native* (libcontainer) driver.
```